minari/data_collector/data_collector.py

```diff
@@ -341,6 +342,7 @@ def create_dataset(
                                                                             `ref_max_score` and `expert_policy` can't be passed at the same time. Default to None
             num_episodes_average_score (int): number of episodes to average over the returns to compute `ref_min_score` and `ref_max_score`. Default to 100.
             minari_version (Optional[str], optional): Minari version specifier compatible with the dataset. If None (default) use the installed Minari version.
+            description (Optional[str], optional): description of the dataset being created. Defaults to None.
```

Comment:
Actually, we should use `(Optional[str]l)`, also the line above is wrong; can you fix it?

Reply:
Do you mean wanting the description to be required instead of defaulting to None?

Reply:
No, it is just for the documentation `Optional[str]l` means it is optional already, so we can remove `, optional`

Reply:
I see throughout the repo many of the other optional args also have the same format of `(Optional[<dtype>], optional)` in docstrings. Should these be kept for consistency?

Reply:
Yes, actually, we should standardize it across all the documentation, so let's do it in another PR.